docs/iris/src/userguide/loading_iris_cubes.rst

```diff
@@ -257,7 +255,6 @@ explicitly enabled by setting the "cell_datetime_objects" option in :class:`iris
     DimCoord([2009-11-19 10:00:00, 2009-11-19 11:00:00, 2009-11-19 12:00:00], standard_name='time', calendar='gregorian')
     >>> # Define a function which accepts a datetime as its argument (this is simplified in later examples).
     >>> hour_11 = iris.Constraint(time=lambda cell: cell.point.hour == 11)
-    >>> with iris.FUTURE.context(cell_datetime_objects=True):
```

Comment:
Again, the following line must be unindented.

Reply:
The two lines beginning with `...` need to be replaced with `>>> cube_11 = cube_all.extract(hour_11)`